composer.json

```diff
@@ -16,7 +16,8 @@
     "prefer-stable": true,
     "require": {
         "firebase/php-jwt": "^6.3",
-        "php": "^8.0"
+        "php": "^8.0",
+        "ext-openssl": "*"
```

Comment:
Are you sure is this needed ?

if so , please,  use a specific version.

Reply:
Yes. The library depends on openssl extension and it is good to specify the requirement so consumers would be aware of it. I cannot be sure of the minimum required version of the extension for this particular library, but even existence of this line should be enough to force consumers installing one of the modern versions.

Reply:
@robsontenorio , I am not sure if this is the holding part for the PR, but I can remove the line if it bothers you to speed up the release. We are going to release a feature depending on this repository in our project, but cannot make tests pass unless the library allows configuring the test token payload. Therefore, your efforts in merging this PR quickly will be greatly appreciated.

Also I am not sure if it is you or @tuytoosh the reviewer as you are the one assigned for this PR.